RocketChat/Rocket.Chat.ReactNative
# [FIX] Condensed layout cutting text on smaller text sizes

## Proposed changes

Removed `condensedPaddingVertical` and a few unnecessary line heights.

## Issue(s)	


## How to test or reproduce

- On iOS Settings (OS App), go to Settings > Display & Brightness > Text Size > Set to minimum
- (It varies on Android, but you might find it on Accessibility)
- Close and reopen the app
- Go to Display > Check "Condensed"
- Go back to rooms list and the text should be small as on screenshots, but it should be rendered correctly.

## Screenshots
<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</thead>
</tr>
<tbody>
<tr>
	<td>
		<img src="https://user-images.githubusercontent.com/804994/156782963-9546b2a1-6386-49f3-8258-2f7de54bc72e.png" width="350" />
	</td>
	<td>
		
<img src="https://user-images.githubusercontent.com/804994/156782939-2a50b628-6c54-4929-b7d6-57aeb8c97cb7.png" width="350" />
	</td>
</tr>

<tr>
	<td>
		<img src="https://user-images.githubusercontent.com/804994/156782955-6ef23226-279e-4652-bdcd-8d4b6a525f9b.png" width="350" />
	</td>
	<td>
		<img src="https://user-images.githubusercontent.com/804994/156782951-47bc65df-9ec6-48f9-803e-ac5e22badd09.png" width="350" />
	</td>
</tr>
</tbody>
</table>





## Types of changes




## Checklist



## Further comments
